scripts/setup-linter-dependencies.py

```diff
@@ -70,7 +70,7 @@ def __init__(self):
         self._memory_store = MemoryStore(stix_data=stix_json["objects"])
 
     @staticmethod
-    def _remove_deprecated_objetcs(stix_objects) -> List[AttackPattern]:
+    def _remove_deprecated_objects(stix_objects) -> List[AttackPattern]:
```

Comment:
This appears to be a private method, so renaming it should be harmless.

(FWIW, the history of `deprecated` is itself amusing, but not relevant here.)